src/courses/mappers/02.md

```diff
@@ -0,0 +1,128 @@
+
+[OASIS Open](https://www.oasis-open.org/) is an international standards body that works on the development and advancement of open source technological standards. OHDF is currently in the process of becoming an OASIS Open standard. For more information on the OHDF charter for OASIS Open, [refer here](https://groups.oasis-open.org/communities/tc-community-home2?CommunityKey=f8888caa-8401-46f8-bf10-018dc7d3f577).
+
+## Why OHDF?
+
+- Many security tools do not provide context to relevant compliance standards for comparison across security tools.​
+- Security tools typically generate data in unique formats that require multiple dashboards and utilities to process.​
+- OHDF reduces the time it takes to process security assessments, data in disparate locations, and inconsistent semantics of a data element between formats.​
+
+## Features
+
+### 1. Consistent integration, aggregation, and analysis of security data from all available sources.​
+
+- Enforces consistent schema fields through consciously designed data format mappings.
+- Supports data format conversion from numerous established security tools such as AWS Security Hub's AWS Security Finding Format (ASFF) and Tenable Nessus' Nessus file format.
+- Allows the integration of currently unsupported security tool data formats through the development of OHDF mappers for the OHDF Converters tool.
+
+![](../../assets/img/OHDF_Inputs.png)
+
+### 2. Preserving data integrity with original source data.
+
+- Uses mappings which maximize meaningful schema field conversions.
+- Leverages schema fields `passthrough` and `raw` to preserve the original data in its entirety.
+- Allows for bidirectional format conversions to and from OHDF.
+
+See below for an example of how *some* fields in a source file are mapped to OHDF (not all the mappings are pictured, for the sake of not cluttering the figure).
+
+![Example of some mappings between a source file and OHDF](../../assets/img/ExampleSchemaMappings.png)
+
+### 3. Maximizing interoperability and data sharing.​
+
+- Provides a consistent and standardized format for communication.
+- Provides an easily ingestible data format and tools to improve user readability.
+
+### 4. Facilitating the transformation and transport of data between security/management processes or technologies.​
+
+- Provides a clear schema for technologies/processes to support.
+- Includes a simple file format that technologies/processes can accept.
+- Compatible with [Heimdall](./03.md#what-is-heimdall) to provide data visualization.
+
+### 5. Allowing for the mapping and enrichment of security data to relevant compliance standards (GDPR, NIST SP 800-53, CCIs, PCI-DSS, etc.).
+
+- Uses mappers which provide and append relevant compliance standards to converted security tool data.
```

Comment:
change: Uses mappers which provide and append relevant **compliance** standards to converted security tool data.
to: Uses mappers which provide and append relevant **categorization** standards to converted security tool data.

Reply:
Addressed.